src/pages/settings/Profile/CustomStatus/StatusPage.tsx

```diff
@@ -231,6 +241,39 @@ function StatusPage() {
                         />
                     )}
                 </View>
+                <View style={[styles.mb2, styles.mt6]}>
+                    <Text style={[styles.mh5]}>{translate('statusPage.setVacationDelegate')}</Text>
+                    {hasVacationDelegate && <Text style={[styles.mh5, styles.mt6, styles.mutedTextLabel]}>{translate('statusPage.vacationDelegate')}</Text>}
+                    {hasVacationDelegate ? (
+                        <OfflineWithFeedback
+                            pendingAction={vacationDelegate?.pendingAction}
+                            errors={vacationDelegate?.errors}
+                            errorRowStyles={styles.mh5}
+                            onClose={clearVacationDelegateError}
+                        >
+                            <MenuItem
+                                title={vacationDelegatePersonalDetails?.displayName ?? formattedDelegateLogin}
```

Comment:
We already have a fallback

https://github.com/user-attachments/assets/dea12208-6fd8-4706-9868-7fd4e9d4f15e